.gitignore

```diff
@@ -3,8 +3,7 @@ Gemfile.lock
 Gemfile.local
 vendor/
 .vendor/
-spec/fixtures/manifests/
-spec/fixtures/modules/
```

Comment:
This comes from modulesync_config and this is on purpose. Some modules ship files as fixtures. For example https://github.com/voxpupuli/puppet-prometheus/tree/master/spec/fixtures/files

Reply:
This now creates the tmpdir inside the fixtures so it needs a change for that at least.